### ISSUE TITLE
Feature Request: Read list of domains from a file

### ISSUE THREAD

**Issue by da667** [bbb31/slurp#8]
Hi there! I love your project and I love that you provide the option to hunt through certstream and/or manual domain specification.

This isn't an issue per se, but a feature request to add in the ability to read a list of domains in for bucket enumeration from a text file.

Thank you for your time and consideration.

**Comment by bbb31** [comment]
```
doms=("google.com" "facebook.com" "github.com"); for i in "${doms[@]}"; do ./slurp domain --domain $i; done
```

for now

**Comment by bbb31** [comment]
This has been fixed in 2.0
